lib/hackney/pdf/preview_generator.rb

```diff
@@ -54,7 +56,11 @@ def get_date
       end
 
       def get_return_date
-        (Time.now + 31.days).strftime('%d %B %Y')
+        (Time.now + 30.days).strftime('%d %B %Y')
```

Comment:
What do you think of this?

```suggestion
        30.days.from_now.strftime('%d %B %Y')
```